Origami Build Tools wont install with the latest version of npm

[But it looks like a problem with gulp-lintspaces]

```
npm WARN package.json fetchres@1.1.0 No repository field.
npm WARN package.json ft-next-express@7.9.2 No repository field.
npm WARN package.json next-ft-api-client@2.1.0 No repository field.
npm WARN prefer global nodemon@1.3.7 should be installed with -g
npm ERR! Darwin 14.3.0
npm ERR! argv "node" "/usr/local/bin/npm" "install"
npm ERR! node v0.12.0
npm ERR! npm  v2.7.6
npm ERR! path /Users/harry.spriggs/next/next-grumman/node_modules/origami-build-tools/node_modules/gulp-lintspaces/node_modules/lintspaces/node_modules/editorconfig/bin\editorconfig
npm ERR! code ENOENT
npm ERR! errno -2

npm ERR! enoent ENOENT, chmod '/Users/harry.spriggs/next/next-grumman/node_modules/origami-build-tools/node_modules/gulp-lintspaces/node_modules/lintspaces/node_modules/editorconfig/bin\editorconfig'
npm ERR! enoent This is most likely not a problem with npm itself
npm ERR! enoent and is related to npm not being able to find a file.
npm ERR! enoent

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/harry.spriggs/next/next-grumman/npm-debug.log
```

Doing `npm install -g npm@2.1.18` (a known working version of npm fixes the issue).  Probably not necessary to fix in OBT as such but if you had a chance to take a look maybe we could fix it in the dodgy dependency ……

[@matthew-andrews actually writing this under Harry's account]

--- a/README.md
+++ b/README.md
@@ -14,8 +14,6 @@ If you have any issues with OBT, please check out [troubleshooting guide](https:
 
 		npm install -g origami-build-tools
 
-__Attention:__ _OBT doesn't work with 'npm >2.7.5' due to a a problem with npm and Editorconfig 0.11.x_
-
 ## Usage
 
 Run the install task for the first time will to install required dependencies:

--- a/TROUBLESHOOT.md
+++ b/TROUBLESHOOT.md
@@ -4,6 +4,8 @@ Different common issues encountered when using OBT organized by the different ta
 
 ### Error `npm ERR! argv "node" "/usr/local/bin/npm" "install" "-g" "origami-build-tools"`
 
+> This error should be fixed in v3.0.2 of origami-build-tools, but we're keeping it in this troubleshooting guide for now.
+
 When install origami-build-tools, depending on the version of [npm](https://github.com/npm/npm), it might fail with an error similar to this:
 
 >npm ERR! Darwin 14.3.0

--- a/package.json
+++ b/package.json
@@ -9,7 +9,7 @@
   },
   "engines": {
     "node": ">= 0.12.2",
-    "npm": ">= 2.0.0<=2.7.5"
+    "npm": ">= 2.0.0"
   },
   "scripts": {
     "test": "mocha -t 500000 `find . -name '*.test.js' -path './test/*'`"

--- a/package.json
+++ b/package.json
@@ -28,7 +28,7 @@
     "gulp-autoprefixer": "^2.1.0",
     "gulp-if-else": "^1.0.3",
     "gulp-jshint": "^1.8.4",
-    "gulp-lintspaces": "^0.2.3",
+    "gulp-lintspaces": "^0.3.0",
     "gulp-minify-css": "^1.0.0",
     "gulp-mustache": "^1.0.2",
     "gulp-rename": "^1.2.0",

--- a/package.json
+++ b/package.json
@@ -44,7 +44,6 @@
     "merge-stream": "^0.1.6",
     "minimist": "^1.1.1",
     "nightwatch": "^0.5.36",
-    "node-sass": "^3.0.0",
     "node.extend": "^1.0.9",
     "npmconf": "^2.1.1",
     "portfinder": "^0.4.0",

